docs/en/modules/tmr.md

```diff
@@ -34,7 +34,12 @@ To free up the resources with this timer when done using it, call [`tmr.unregist
 
 #### Example
 ```lua
-if not tmr.alarm(0, 5000, tmr.ALARM_SINGLE, function() print("hey there") end) then print("whoopsie") end
+if not tmr.create():alarm(5000, tmr.ALARM_SINGLE, function()
```

Comment:
I like the `tmr.create():alarm()` thing - looks like a nice single-command substitution. But haven't actually tried it myself. Does it survive the next garbage collection before the timer expires?